python/tvm/driver/tvmc/compiler.py

```diff
@@ -442,6 +456,11 @@ def compile_model(
         if dumps:
             save_dumps(package_path, dumps)
 
+        # Print compilation times per pass
+        if timing_inst:
+            print("Printing results of timing profile...")
```

Comment:
Perhaps we could rename to "Compilation time breakdown by pass:" - I feel like it's a bit more informative